package.json

```diff
@@ -8,7 +8,7 @@
     "DSL"
   ],
   "scripts": {
-    "test": "mocha test/*"
+    "test": "mocha test/reader/*"
```

Comment:
In that case we need to move the test_files location as mocha fails trying to find an index.js there